pyccel/ast/bind_c.py

```diff
@@ -553,12 +553,15 @@ class BindCClassDef(ClassDef):
     ----------
     original_class : ClassDef
         The class being wrapped.
+
+    **kwargs : dict
+        See ClassDef.
     """
     __slots__ = ('_original_class',)
 
-    def __init__(self, original_class):
+    def __init__(self, original_class, **kwargs):
         self._original_class = original_class
-        super().__init__(original_class.name)
+        super().__init__(original_class.name, **kwargs)
```

Comment:
Pyccel Bot complains that `BindCClassDef` is not part of `__all__`

Reply:
Fixed [d7d71f7](https://github.com/pyccel/pyccel/pull/1620/commits/d7d71f7d00edd6eed967ad3c5874f1aca51f4018)

Reply:
Fixed in d7d71f7